cmd/completion.go

```diff
@@ -0,0 +1,280 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/qri-io/ioes"
+	"github.com/spf13/cobra"
+)
+
+// NewAutocompleteCommand creates a new `qri complete` cobra command that prints autocomplete scripts
+func NewAutocompleteCommand(_ Factory, ioStreams ioes.IOStreams) *cobra.Command {
+	o := &AutocompleteOptions{IOStreams: ioStreams}
+	cmd := &cobra.Command{
+		Use:   "completion [bash|zsh]",
+		Short: "Generates shell completion scripts",
```

Comment:
keeping with our [CLI Help Style](https://github.com/qri-io/qri/blob/master/DEVELOPERS.md#cli-help-style), should be `generate shell auto-completion scripts`